django_bootstrap_dynamic_formsets/templates/django_bootstrap_dynamic_formsets/dynamic_formsets_js.html

```diff
@@ -1,171 +1,179 @@
+                    return false;
+                }
+            });
+            return isEmpty;
+        };
+
+
+        //Writes the correct current order in the order form fields
+        var correctOrder = function () {
+            var localSortItemCounter = 1;
+            $("[id$=ORDER]").each(function () {
+                $(this).attr('value', localSortItemCounter);
+                localSortItemCounter++;
+            });
+        };
+
+        //Hide "Order" and "Delete" field/checkbox
+        $("[id$=ORDER]").parents(".form-group").hide();
+        $("[id$=DELETE]").parents(".form-group").hide();
+
+        //Holds an empty form
+        var $emptyForm = $('#{% if formset.prefix %}{{formset.prefix}}-{% endif %}empty-form').remove().children().unwrap();
+
+        var $sortItem = $('.sort-item.{{formset.prefix}}');
+
+        //Disable Up button on first and Down button on last form
+        var setUpDownButtons = function () {
+            $sortItem = $('.sort-item.{{formset.prefix}}');
+            $sortItem.find(".up-form{% if formset.prefix %}-{{formset.prefix}}{% endif %}, .down-form{% if formset.prefix %}-{{formset.prefix}}{% endif %}").each(function () {
+                $(this).prop("disabled", false);
+            });
+            $sortItem.first().find(".up-form{% if formset.prefix %}-{{formset.prefix}}{% endif %}").prop("disabled", true);
+            $sortItem.last().find(".down-form{% if formset.prefix %}-{{formset.prefix}}{% endif %}").prop("disabled", true);
+        };
+
+
+        //Hide forms that are marked for deletion
+        $sortItem.find('[id$=DELETE]').each(function () {
+            if ($(this).prop('checked')) {
+                $(this).parents(".sort-item").hide();
+            }
+        });
+
+
+        //Counts the number of forms
+        var sortItemCounter = 0;
+        $sortItem.each(function () {
+            sortItemCounter++;
+        });
+        //$('body').prepend('<p>Count ({{formset.prefix}}): ' + sortItemCounter + '</p>');
+
+        //Mark initial forms as such
+        var numInitialForms = $('#id_{% if formset.prefix %}{{formset.prefix}}-{% endif %}INITIAL_FORMS').val();
+        var i = 0;
+        $sortItem.each(function () {
+            if (i < numInitialForms) {
+                $(this).addClass("initial-form");
+                i++;
+            } else {
+                return false;
+            }
+        });
+
+        var numTotalForms = $('#id_{% if formset.prefix %}{{formset.prefix}}-{% endif %}TOTAL_FORMS').val();
+        //Fix order (important if page reloads after validation failed)
+        for (i = 1; i < numTotalForms; i++) {
+            $sortItem.find('[id$=ORDER][value=' + (i + 1) + ']').parents(".sort-item").insertAfter($sortItem.find('[id$=ORDER][value=' + i + ']').parents(".sort-item"));
+        }
+
+
+        //Set up sortable jQuery UI interaction
+        {% if can_order %}
+
+            var $sortable = $('#sortable');

```

Comment:
I'm not sure if this line could cause problems with multiple formsets in a page. If I'm not mistaken you would have more than one element with this id `sortable` on your page.